pages/reporting/details/deadlines/views/index.jsx

```diff
@@ -18,14 +22,22 @@ export default function Deadlines() {
         </div>
       </div>
 
-      <MetricsFilter start={start} end={end} filterEstablishment={false} />
+      <MetricsFilter start={startDate} end={end} filterEstablishment={false} setStartDate={setStartDate} />
+      {
+        format(startDate, 'YYYY-MM-DD') < '2022-04-01' ? <Warning>Missed internal deadline data is not available before 1 April 2022</Warning> : ''
+      }
 
       <div className="govuk-grid-row">
         <div className="govuk-grid-column-one-half">
           <Metric number={deadlines} label="Missed statutory deadline" />
         </div>
         <div className="govuk-grid-column-one-half">
-          <Metric number={internalDeadlines.total} label="Missed internal target" />
+          {
+            end < '2022-02-01' ? <div className="metric">
```

Comment:
Just to help anyone coming back to this file in the future, it might be worth encapsulating `end < '2022-02-01'` as a named var so it's immediately obvious what the purpose of that specific date is, e.g.

```const isBeforeInternalDeadlines = end < '2022-02-01;```

Reply:
Or at least just define the date once, with a comment as to why that date is pertinent.

```
// data for internal deadlines was not collected before February 2022
```

Reply:
Have amended

Reply:
Thanks!